src/components/BreakoutRoomsEditor/SelectableParticipant.vue

```diff
@@ -4,35 +4,47 @@
 -->
 
 <template>
-	<div tabindex="0"
-		class="selectable-participant">
-		<input id="participant.attendeeId"
+	<label :for="participant.attendeeId"
+		class="selectable-participant"
+		:class="{ 'selectable-participant--selected': selected }">
+		<input :id="participant.attendeeId"
 			v-model="modelProxy"
 			:value="participant.attendeeId"
 			type="checkbox"
-			name="participant.attendeeId">
+			class="hidden-visually">
```

Comment:
We probably need a new class here, but this `hidden-visually` is not valid for a11y for interactive element. It is only valid for visually hidden text description.

It makes an input a part of the page but located in a random far place. Side effects:
- When using virtual cursor from assistive technologies, it may move cursor to the actual checkbox position on the page (completely random place)
- When focusing to the element it may:
  - if we are lucky and we are in a scrollable container - just don't scroll
  - if we are not and we are not in a scroll non-scrollable container - scroll the container and broke it
  
To reproduce:
1. Have many users
2. Make it small
3. Move via Tab

Expected - scrolls moves with you

Actual - it only moves when we focus 

Actual | Expected
---|---
![bug](https://github.com/user-attachments/assets/064620ff-432e-42ff-8fd1-b0f74cf0db79) | ![no-bug](https://github.com/user-attachments/assets/28eeabe0-7dbe-43dd-8a9c-554da54cacaf)

Reply:
A correct solution is to create a new class with similar styles, but:
- Make `label`: `position: relative`, so the `input` is inside the label
- Don't make `input` moved away with `top: 10000px`, just place it somewhere in the list item in a meaningful place
- Add `z-index: -1` to visually hide it.

Additionally we could make it reasonable size in a specific position to mimic real checkbox, but in this element there is no place for it. No visual element that looks like a checkbox.

To, putting it in the `label` is enough I'd say.